geometry_msgs/PoseStamped received_pose
string frame_name
---
geometry_msgs/PoseStamped transformed_pose
bool succeed
